src/Apps/Sell/Routes/SubmissionRoute.tsx

```diff
@@ -28,7 +28,10 @@ export const SubmissionRoute: React.FC<SubmissionRouteProps> = props => {
       <AppContainer>
         <SellMeta />
 
-        <SellFlowContextProvider submissionID={submission.externalId}>
+        <SellFlowContextProvider
+          submissionID={submission.externalId}
+          internalSubmissionID={submission.internalID}
```

Comment:
Instead of passing two props, we could only use one `submission` prop (thanks @dblandin https://github.com/artsy/force/pull/14182#discussion_r1672301184)

Reply:
oki

Reply:
updated it.